### PR TITLE
FINERACT-1857: Backport Flyway migrations to Liquibase for 1.8.x

### PR DESCRIPTION
**Pull request by vidakovic**
Backport FINERACT-1846 for 1.8.x releases.

### CODE REVIEW

File: fineract-provider/src/main/resources/db/changelog/tenant-store/upgrades/0000_upgrade_to_1.6.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    KIND, either express or implied. See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <!-- V5__add_schema_connection_parameters.sql -->
+    <changeSet author="fineract" id="5">
```
Comment (adamsaghy) [inline fineract-provider/src/main/resources/db/changelog/tenant-store/upgrades/0000_upgrade_to_1.6.xml:26]: Flyway was setting the below to the default tenant. Is this okay to be left out?
```
UPDATE `tenant_server_connections`
  SET `schema_connection_parameters`='${fineract.tenant.parameters}'
  WHERE `id`=1;
  ```

File: fineract-provider/src/main/resources/db/changelog/tenant/upgrades/0000_upgrade_to_1.5.xml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            <column name="id" valueNumeric="10"/>
+            <column name="hook_template_id" valueNumeric="3"/>
+            <column name="field_type" value="string"/>
+            <column name="field_name" value="Index Name"/>
+            <column name="placeholder"/>
+            <column name="optional" valueBoolean="true"/>
+        </insert>
+    </changeSet>
+    <!-- V365__reportCategoryList-FINERACT-1306.sql -->
+    <changeSet author="fineract" id="365">
```
Comment (adamsaghy) [inline fineract-provider/src/main/resources/db/changelog/tenant/upgrades/0000_upgrade_to_1.5.xml:69]: V365__reportCategoryList-FINERACT-1306.sql looks different in flyway:
the two (stretchy report) tables are not missing?

File: fineract-provider/src/main/resources/db/changelog/tenant-store/upgrades/0000_upgrade_to_1.6.xml (hunk 1 above)
Comment (vidakovic) [reply fineract-provider/src/main/resources/db/changelog/tenant-store/upgrades/0000_upgrade_to_1.6.xml:26]: Note: this is a 1:1 backport of @josehernandezfintecheandomx 's implementation that is already merged in upstream develop. Maybe @josehernandezfintecheandomx can chime in here?

File: fineract-provider/src/main/resources/db/changelog/tenant/upgrades/0000_upgrade_to_1.5.xml (hunk 2 above)
Comment (vidakovic) [reply fineract-provider/src/main/resources/db/changelog/tenant/upgrades/0000_upgrade_to_1.5.xml:69]: Same here... maybe @josehernandezfintecheandomx can shed some light?

**Pull request merged**